go-controller/Makefile

```diff
@@ -24,7 +24,7 @@ install:
 	install -D -m 755 ${OUT_DIR}/go/bin/ovnkube ${BINDIR}/
 	install -D -m 755 ${OUT_DIR}/go/bin/ovn-k8s-overlay ${BINDIR}/
 	install -D -m 755 ${OUT_DIR}/go/bin/ovn-kube-util ${BINDIR}/
-	install -D -m 755 ${OUT_DIR}/go/bin/ovn-k8s-cni-overlay ${CNIBINDIR}/
+	install -D -m 755 ${OUT_DIR}/go/bin/ovn-k8s-cni-overlay -t ${CNIBINDIR}/
```

Comment:
Why do we need '-t' here and not for commands above?

Reply:
Just because the default for BINDIR is /usr/bin and is usually existing in the system. The default for CNIBINDIR is /opt/cni/bin and its never found existing on a new system.
Do you think we should mkdir (what -t does) the other destination directories too?

Reply:
I did not know that '-t' does a mkdir. The man page does not seem to have that information.